messaging/src/main/java/org/axonframework/eventhandling/tokenstore/jpa/JpaTokenStore.java

```diff
@@ -49,6 +57,8 @@
 public class JpaTokenStore implements TokenStore {
 
     private static final Logger logger = LoggerFactory.getLogger(JpaTokenStore.class);
+    private static final String CONFIG_TOKEN_ID = "__config";
```

Comment:
Just dropping a thought here: Would it make sense to have these constants in the `TokenStore` interface?

Reply:
That crossed my mind, actually. But the approach taken here using a ConfigToken is merely an implementation detail for relational databases. The fact that both JDBC and JPA use the same name is "coincidental" 😉 .